tests/test_meshing_workflow.py

```diff
@@ -709,7 +709,7 @@ def test_watertight_workflow_dynamic_interface(mixing_elbow_geometry, new_mesh_s
     assert watertight.create_volume_mesh is None
 
 
-@pytest.mark.fluent_version(">=23.2")
+@pytest.mark.fluent_version("==23.2")
```

Comment:
https://github.com/ansys/pyfluent/issues/1926

Reply:
@mkundu1 Cool, I will close it now...